pg_search/src/postgres/customscan/aggregatescan/scan_state.rs

```diff
@@ -106,6 +129,8 @@ impl AggregateScanState {
                 "field".to_string(),
                 serde_json::Value::String(group_col.field_name.clone()),
             );
+            // if we remove this, we'd get the default size of 10, which means we receive 10 groups max from tantivy
+            terms.insert("size".to_string(), serde_json::Value::Number(10000.into())); // TODO: make configurable
```

Comment:
I dropped this in a previous PR and made me crazy until found out the hard way that we need to set it. We should still find the best way to set this. Not an issue to handle in this PR though.

Reply:
Added a separate issue to track this: https://github.com/paradedb/paradedb/issues/2964 and also modified the comment in here.